CONTRIBUTING_DESIGN.md

```diff
@@ -0,0 +1,200 @@
+# `/docs` & 11ty documentation
+
+The Red Hat Digital Design System site is built with [Eleventy](https://www.11ty.io/).
+
+## Installation
+Make sure you have at least Node 16 and run:
+
+```shell
+npm ci
```

Comment:
Do we need a node about changing your directory to the local design-system repo?